Makefile.am

```diff
@@ -3794,6 +3794,7 @@ endif
 if HAVE_GNUTLS
 systemd_journal_remote_LDADD += \
 	$(GNUTLS_LIBS)
+endif
 
 # systemd-journal-remote make sense mostly with full crypto stack
```

Comment:
unfortunately there seem to be 'concerns' about shipping gnutls (https://github.com/coreos/coreos-overlay/pull/1587#issuecomment-148422613), and i believe that the unit files and directory for remote journal should be created regardless of tls support.